### ISSUE TITLE
Display server invites like it's done in Discord

### ISSUE THREAD

**Issue by CanePlayz** [Tyrrrz/DiscordChatExporter#649]
### Export format

HTML

### Details

I have no clue if that's even doable with the data you can get from the Discord API and so on, but my idea here was to display the preview for server invites a bit more like it's done in Discord itself.

(for reference, here's how it looks on Discord)
![image](https://user-images.githubusercontent.com/81419612/125693919-37855849-29c3-4454-88af-a10ebbcd94bd.png)

(and in the HTML file it's just blank currently)
![image](https://user-images.githubusercontent.com/81419612/125693985-34e7e715-d2b5-44bc-a348-c45f97cb9568.png)

(the link obviously doesn't work anymore xD)

**Comment by Tyrrrz** [comment]
Yeah, not sure if there's any detailed information on the invite coming from the API. Will have to investigate.

**Comment by Fyko** [comment]
@Tyrrrz the `/invites/{invite_code}` endpoint exists
https://discord.com/api/invites/djs?with_counts=true

**Comment by Tyrrrz** [comment]
> @Tyrrrz the `/invites/{invite_code}` endpoint exists
> [discord.com/api/invites/djs?with_counts=true](https://discord.com/api/invites/djs?with_counts=true)

Nice!

**Comment by Tyrrrz** [comment]
Added basic support for invite links, where it renders the guild icon, guild name, and channel name. Discord client currently renders online/total member count instead of channel name, but I feel like the former is more useful in an export.

**Comment by CanePlayz** [comment]
Great, thanks. 👍
